src/diet.ts

```diff
@@ -328,11 +329,18 @@ function menu(): MenuItem<Note>[] {
   const boxingDayCareItems = $items`glass of raw eggs, punch-drunk punch`.filter((item) =>
     have(item)
   );
-  const pilsners = $items`astral pilsner`.filter((item) => globalOptions.ascend && have(item));
+  const pilsners = $items`astral pilsner`.filter((item) => have(item));
   const limitedItems = [...boxingDayCareItems, ...pilsners].map(
     (item) => new MenuItem<Note>(item, { maximum: availableAmount(item) })
   );
 
+  const legendaryPizzas = [
+    { pizza: $item`Pizza of Legend`, pref: "pizzaOfLegendEaten" },
+    { pizza: $item`Calzone of Legend`, pref: "calzoneOfLegendEaten" },
+  ]
+    .filter(({ pref }) => !get(pref))
```

Comment:
we want to check whether or not you have all the recipes, much like we do in the `ofLegendPotion` function